src/test/java/org/gridsuite/modification/server/modifications/TwoWindingsTransformerModificationTest.java

```diff
@@ -576,6 +577,215 @@ public void testPhaseTapChangerModification() throws Exception {
+
+        phaseTapChanger = getNetwork().getTwoWindingsTransformer(twtId).getPhaseTapChanger();
+
+        // modification 4 assert
+        assertEquals(PhaseTapChanger.RegulationMode.CURRENT_LIMITER, phaseTapChanger.getRegulationMode());
+        assertEquals(2.0, phaseTapChanger.getTargetDeadband());
+        assertEquals(8.0, phaseTapChanger.getRegulationValue());
+        assertTrue(phaseTapChanger.isRegulating());
+
+        // modification 5 : CURRENT_LIMITER -> FIX_TAP
+        phaseTapChangerCreation.getPhaseTapChanger().setRegulationMode(new AttributeModification<>(PhaseTapChanger.RegulationMode.FIXED_TAP, OperationType.SET));
+        phaseTapChangerCreation.getPhaseTapChanger().setRegulationValue(null);
+        phaseTapChangerCreation.getPhaseTapChanger().setTargetDeadband(null);
+
+        String modificationToModifyJson5 = mapper.writeValueAsString(phaseTapChangerCreation);
+        mockMvc.perform(post(getNetworkModificationUri()).content(modificationToModifyJson5).contentType(MediaType.APPLICATION_JSON))
+            .andExpect(status().isOk()).andReturn();
+
+        phaseTapChanger = getNetwork().getTwoWindingsTransformer(twtId).getPhaseTapChanger();
+
+        // modification 5 assert
+        assertEquals(PhaseTapChanger.RegulationMode.CURRENT_LIMITER, phaseTapChanger.getRegulationMode());
+        assertEquals(2.0, phaseTapChanger.getTargetDeadband());
+        assertEquals(8.0, phaseTapChanger.getRegulationValue());
+        assertFalse(phaseTapChanger.isRegulating());
+    }
+
+    @Test
+    public void testPhaseTapChangerRegulationModification2() throws Exception {
+        TwoWindingsTransformer twt3 = createPhaseTapChanger();
+        String twtId = "trf3";
+
+        // modification 1 : FIXED_TAP -> ACTIVE_POWER_CONTROL error
+        TwoWindingsTransformerModificationInfos phaseTapChangerCreation = TwoWindingsTransformerModificationInfos.builder()
+            .stashed(false)
+            .equipmentId(twtId)
+            .ratioTapChanger(RatioTapChangerModificationInfos.builder()
+                .build())
+            .phaseTapChanger(PhaseTapChangerModificationInfos.builder()
+                .enabled(new AttributeModification<>(true, OperationType.SET))
+                .regulationMode(new AttributeModification<>(PhaseTapChanger.RegulationMode.ACTIVE_POWER_CONTROL, OperationType.SET))
+                .build())
+            .build();
+
+        String modificationToModifyJson1 = mapper.writeValueAsString(phaseTapChangerCreation);
+
+        // modification 1 assert
+        MvcResult result = mockMvc.perform(post(getNetworkModificationUri())
+                .content(modificationToModifyJson1).contentType(MediaType.APPLICATION_JSON))
+            .andExpect(status().isOk()).andReturn();
+        assertEquals("{\"applicationStatus\":\"WITH_ERRORS\",\"lastGroupApplicationStatus\":\"WITH_ERRORS\",\"networkImpacts\":[],\"impactedSubstationsIds\":[]}",
+            result.getResponse().getContentAsString());
```

Comment:
```suggestion
            assertLogMessage(new NetworkModificationException(MODIFY_TWO_WINDINGS_TRANSFORMER_ERROR, "Regulation value is missing, phase tap changer can not regulate").getMessage(),
            phaseTapChangerCreation.getErrorType().name(), reportService);
```

In my opinion, an assert on result is less interesting than an assert on log